### DISCUSSION TITLE
pixi no longer found on windows in GitHub action

### DISCUSSION THREAD

**Discussion by jonmmease** [prefix-dev/pixi#461] [General]
I'm having an issue that I think started after the pixi 0.7.0 release. For VegaFusion, I have GitHub actions that installs pixi on windows with `iwr -useb https://pixi.sh/install.ps1 | iex` in one step and then use it with `pixi run ...` in subsequent steps. This has worked since we first started using pixi, but now I'm getting an error:

(See https://github.com/hex-inc/vegafusion/actions/runs/6877165393/job/18704260177?pr=419)

```
Run pixi run build-py-embed
pixi: D:\a\_temp\9fabafc1-fa89-[4](https://github.com/hex-inc/vegafusion/actions/runs/6877165393/job/18704260177?pr=419#step:5:5)704-af7[5](https://github.com/hex-inc/vegafusion/actions/runs/6877165393/job/18704260177?pr=419#step:5:6)-505ec9dbef45.ps1:2
Line |
   2 |  pixi run build-py-embed
     |  ~~~~
     | The term 'pixi' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the
     | spelling of the name, or if a path was included, verify that the path is correct and try again.
```

The installation in the previous step appears to be succeeding:

```
Run iwr -useb https://pixi.sh/install.ps1 | iex
This script will automatically download and install Pixi (latest) for you.
Getting it from this url: https://github.com/prefix-dev/pixi/releases/latest/download/pixi-x86_64-pc-windows-msvc.zip
The binary will be installed into 'C:\Users\runneradmin\.pixi\bin'

    Directory: C:\Users\runneradmin\.pixi

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          11/1[5](https://github.com/hex-inc/vegafusion/actions/runs/6877165393/job/18704260177?pr=419#step:3:6)/2023 12:1[8](https://github.com/hex-inc/vegafusion/actions/runs/6877165393/job/18704260177?pr=419#step:3:9) PM                bin
Adding C:\Users\runneradmin\.pixi\bin to PATH
```

I won't have time to dig into the issue for a few days, but wanted to ask if something changed that might explain this.

Also, what's the best way to pin the version of pixi for use in a CI context?

**Comment by ruben-arts** [comment]
That is indeed a problem that was introduced by merging a fix in the installation script. 

We're not installing in `AppData` anymore, but in `$USERPROFILE\.pixi\bin` which is the same as on Unix. 

The best tip to use pixi in CI is using the https://github.com/prefix-dev/setup-pixi action as this will all be maintained and fixes like this will be fixed first in that action.

**Reply by jonmmease** [reply]
Thanks for the response! I didn't know about the `setup-pixi` action. I'll try migrating to that
